### DISCUSSION TITLE
Allow disabling environment variable encryption

### DISCUSSION THREAD

**Discussion by jacobdo2** [vercel/vercel#5240] [Ideas] [answered]
I have a nextjs project deployed on vercel across multiple environments and storing my environment variables for production under production, while for staging, development and prs - I am (trying) to store them under preview by prefixing each value of the environment that it belongs to e.g. STAGING_ ... , DEVELOPMENT_ ... , PR_ ... . 

We all make mistakes and I have, on numerous occasions, entered the wrong value for one of the keys in the given environments and I have no other way of checking the values than adding some additional logic in my application to actually output these values either during build step or in the client, which is dangerous, but also quite cumbersome. 

Would you consider adding an option that would allow disabling encryption for environment variables?

**Comment by styfle** [comment]
Hi @jacobdo2 

We plan to add a new feature that lets you decide if you are adding a plain text env var or an encrypted env var.

We'll let you know once it is available in the dashboard, thanks!

**Comment by lucleray** [comment, marked as answer]
Hi @jacobdo2,

I'm happy to share with you that this is now possible with [Plaintext Environment Variables](https://vercel.com/docs/platform/environment-variables#plaintext-environment-variables).

Let us know if it solves your use case.

**Reply by jacobdo2** [reply]
Fantastic, thank you very much.
